### PR TITLE
Handle failed auth

### PR DESCRIPTION
**Pull request by Jake-Lindvall**
Using `axios` interceptors and some component restructuring to display error splashes, particularly related to auth concerns

NOTE: keep in mind for future development: any and all error codes encountered that match ones specified in `<Navigation>`'s markup will kick off the interceptor and take over the view. More complex logic may be required if we like this approach but find conflicts in handling small, view-specific errors vs these "global" ones.

Does not necessarily need to be merged, but am looking for opinions!

### CODE REVIEW

File: src/Components/Scenes/Chrome/Navigation/Navigation.js [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+    );
+  });
+
+  // Case 1: there is a SAML response but no session token, so authenticate real quick
+  // Case 2: there is neither a SAML response nor a session token, so redirect to OneLogin
+  // Case 3: there may or may not be a SAML response,
+  //         but there is a session token, so relax until an hour later
+  //         when we get a 4xx code from some request, then redirect
+  const samlResponse = useQuery('SAMLResponse');
+  const [isRedirecting, setIsRedirecting] = useState(true);
```
Comment (ghost) [inline src/Components/Scenes/Chrome/Navigation/Navigation.js:96]: nitpick: just the naming isRedirecting implies this should be true when redirecting to OneLogin. Maybe something to indicate that the API returned an error code like `errorPage`

**Review by ghost — COMMENTED** [review]
LGTM overall! small small nitpick. 

Also to be clear, this doesn't handle undefined routes at the moment, and that would be an add in the future (prob when the designs for the error states are confirmed) correct?

File: src/Components/Scenes/Chrome/Navigation/Navigation.js (hunk 1 above)
Comment (Jake-Lindvall) [reply src/Components/Scenes/Chrome/Navigation/Navigation.js:96]: Let's sync up on this but I believe it is set to true when it does redirect - the case here is to detect the "first" time a user lands, not to auto-redirect on something like an `errorPage` encountered. That view (on a `401` or `403`) will have a manual link out to OneLogin if Scott's designs stay relatively the same :)

Comment (ghost) [reply src/Components/Scenes/Chrome/Navigation/Navigation.js:96]: Per our convo Friday I am good with the naming here

**Review by ghost — DISMISSED** [review]

**Review by SelcukEdma — DISMISSED** [review]
looks good need to solve mr conflict

**Review by davidkliewer-kirby — DISMISSED** [review]

**Review by ghost — APPROVED** [review]
fixed import path but LGTM

**Review by SelcukEdma — APPROVED** [review]

**Pull request merged**